lib/logstash/outputs/lumberjack.rb

```diff
@@ -73,6 +74,10 @@ def flush(events, close = false)
       retry
     end # begin
   end
+
+  def close
+    buffer_flush(:force => true) 
```

Comment:
Also the ruby-lumberjack doesnt support close at the moment, but I believe this to not be an important issue issue to fix right now. created https://github.com/elastic/ruby-lumberjack/issues/21